Lib/test/test_curses.py

```diff
@@ -1095,6 +1095,16 @@ def test_resizeterm(self):
         self.assertEqual(curses.LINES, lines)
         self.assertEqual(curses.COLS, cols)
 
+        for func in (curses.resizeterm, curses.resize_term):
```

Comment:
Note that `resize_term` is not always available on all platforms.

Reply:
It's fine here though, right? I'm not seeing anything about availability in the docs for `resize_term`.

Reply:
The C code is guarded by a `HAVE_CURSES_RESIZE_TERM` so it could not always be available.

Reply:
Oh but there is an issue in the module. It unconditionally exports the method!!!

```C
    _CURSES_RESIZETERM_METHODDEF
    _CURSES_RESIZE_TERM_METHODDEF
```

Reply:
Ok, I just moved them to separate tests. I think that should fix it.

> Oh but there is an issue in the module. It unconditionally exports the method!!!
> 
> ```c
>     _CURSES_RESIZETERM_METHODDEF
>     _CURSES_RESIZE_TERM_METHODDEF
> ```

Why do we have two functions for this, anyway? I would be fine with soft-deprecating `resize_term` if it has compatibility issues.

Reply:
Because these functions have different purposes (see `man resize_term`)